lib/Remote/Service/DriverService.php

```diff
@@ -110,25 +110,24 @@ public function isRunning()
     }
 
     /**
-     * Check if the executable is executable.
-     *
      * @param string $executable
      * @throws Exception
-     * @return string
      */
-    protected static function checkExecutable($executable)
+    protected function setExecutable($executable)
```

Comment:
Bc break?

Reply:
😕

Reply:
I added the `protected static function checkExecutable($executable)` back. But it just returns the value. However the API is kept, and executable is now checked when calling setExectauble from the constructor.